ui/apps/platform/cypress/helpers/request.js

```diff
@@ -114,3 +120,62 @@ export function interactAndInspectGraphQLVariables(interactionCallback, opname)
         return cy.wrap(interception.request.body.variables);
     });
 }
+
```

Comment:
This is a little verbose, but changing the whole file to TS caused git to track it as a delete and a new file instead of a rename. I may follow up with a proper conversion to TS.